src/Okta.Idx.Sdk/IMessage.cs

```diff
@@ -16,6 +13,9 @@ namespace Okta.Idx.Sdk
     public interface IMessage : IResource
     {
         string Text { get; }
+
         string Class { get; }
```

Comment:
I don't think we should expose the class since that shouldn't be relevant for devs.

Reply:
the Class can be `INFO` or `ERROR`, maybe something else, it's not completely useless. Do you think it's better to hide this info?

Reply:
To me, this seems to be exclusively for the SIW. I think we shouldn't expose anything related to the UI.